Java/core/src/main/java/com/amazon/randomcutforest/CommonUtils.java

```diff
@@ -200,7 +200,7 @@ public static double defaultScalarNormalizerFunction(double scalarValue, double
     };
 
     public static double[] toDoubleArray(float[] point) {
-        checkNotNull(point, "point must not be null");
+        checkNotNull(point, "array must not be null");
```

Comment:
The error message was written to match the formal parameter name, so I would either change this back or change the formal parameter to `array`.

Reply:
Done